galaxy.yml

```diff
@@ -0,0 +1,15 @@
+---
+authors:
+  - TODO
+dependencies:
+  "ansible.netcommon": ">=2.0.1"
+license_file: LICENSE
+name: lxc_ssh
+namespace: ansible
+description: Ansible Network Collection for LXC containers and hosts
+readme: README.md
+repository: https://github.com/my_org/my_collection"
```

Comment:
@andreasscherbaum as an example: I only created the framework, the actual content has to be reviewed and edited. You as current maintainer know best, right? ;-)

Reply:
@stefangweichinger Ah, I actually never created a collection :-( Have to admit that.
How about I leave this until all the other PRs are merged, and then we look over this one as well?

Reply:
Sure, that was also my plan. I just wanted to see if the tests still work and which changes are needed, etc.